src/assets/big-icons/index.js

```diff
@@ -25,15 +25,16 @@ export { default as documentsLockIcon } from './ic_documents-lock.svg';
 export { default as documentsStatisticIcon } from './ic_documents-statistic.svg';
 export { default as elevatorIcon } from './ic_elevator.svg';
 export { default as envelopeMailLetterIcon } from './ic_envelope-mail-letter.svg';
+export { default as euroChangeIcon } from './ic_euro-change.svg';
```

Comment:
You didn't mention this one in the description but it has also been added, I think @homeday-de/realtor-performance-squad-frontend uses it and can replace by blocks afterwards.

Reply:
I'll check the whole changes and write the description again, I'll be pinging the teams to update their stack after this merge.

Reply:
I updated the description.